lib/cloudshaper/cli.rb

```diff
@@ -4,6 +4,7 @@
 module Cloudshaper
   class CLI < Thor
     class_option 'remote_state', type: 'boolean'
+    class_option 'terraform_path', type: "string", default: 'terraform'
```

Comment:
Maybe a small description mentioning that this is the path to the binary (not the path containing the binary)?